### DISCUSSION TITLE
Move CC:T and CC:R Repositories to a github Organization

### DISCUSSION THREAD

**Discussion by Merith-TK** [cc-tweaked/CC-Tweaked#814] [General]
So you know how CC:R and CC:T are under seperate Devs? I propose a unified "originization" here on github, where we can have both our repositories easily accessible to everyone, and even include other mods that help out new comers, such as [Toad-Dev's Example Peripheral for Fabric](https://github.com/Toad-Dev/cc-peripheral-test) and maybe even an example peripheral for CC:T, along wiht various other mods that arnt standalone, but enhance computercraft, 

I propose the name "ComputerCraft United" (because "ComputerCraft" is taken by a almost decade old porting the original CC to the sprout plugin system)

**Comment by Lupus590** [comment]
It's just occurred to me that you could put forks into the org and basically treat them as shortcuts/links. So the org will become a way of discovering active CC mods.

**Reply by Merith-TK** [reply]
Yep! I mentioned this, but i dont think I wrote it perfectly, so yeah!

**Reply by nullified33** [reply]
> So you know how CC:R and CC:T are under seperate Devs? I propose a unified "originization" here on github, where we can have both our repositories easily accessible to everyone, and even include other mods that help out new comers, such as [Toad-Dev's Example Peripheral for Fabric](https://github.com/Toad-Dev/cc-peripheral-test) and maybe even an example peripheral for CC:T, along wiht various other mods that arnt standalone, but enhance computercraft,
> 
> I propose the name "ComputerCraft United" (because "ComputerCraft" is taken by a almost decade old porting the original CC to the sprout plugin system)

No offense to Merith-TK but I think the name CC:United is lackluster and the organization should instead be called CC: Continued or CC: Revival or CC-Repos

**Reply by Merith-TK** [reply]
> > So you know how CC:R and CC:T are under seperate Devs? I propose a unified "originization" here on github, where we can have both our repositories easily accessible to everyone, and even include other mods that help out new comers, such as [Toad-Dev's Example Peripheral for Fabric](https://github.com/Toad-Dev/cc-peripheral-test) and maybe even an example peripheral for CC:T, along wiht various other mods that arnt standalone, but enhance computercraft,
> > I propose the name "ComputerCraft United" (because "ComputerCraft" is taken by a almost decade old porting the original CC to the sprout plugin system)
> 
> No offense to Merith-TK but I think the name CC:United is lackluster and the organization should instead be called CC: Continued or CC: Revival or CC-Repos

Hey side tip, you can make a seperate comment for this thread at the bottom

and I 100% agree that CC:United shouldnt be the finallized name for the organization,

**Comment by SirEndii** [comment]
I don't really know, if CC really needs an organization. 

But if there would be an organization, you could add some big cc integration mods to this organization 😉

**Reply by Lupus590** [reply]
Or lots of smaller integration mods.

**Comment by SquidDev** [comment]
[Done](https://github.com/cc-tweaked/CC-Tweaked/)
